src/App.js

```diff
@@ -1,5 +1,41 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+import { MESSAGES } from "../src/constants/messages.js";
+import carInputValidation from "./utils/carInputValidation.js";
```

Comment:
```suggestion
import { MESSAGES } from "./constants/messages.js";
import carInputValidation from "./utils/carInputValidation.js";
```
`MESSAGES`를 import 하는 경로가 아래 부분이랑은 다르게 루트로 갔다가 다시 경로를 찾은 것 같은데 이렇게 작성하신 이유가 있으신가요??

Reply:
앗 실수했네요 ! 고치겠습니다